src/Components/Components/src/Rendering/InvalidEventException.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Components.Rendering
+{
+    /// <summary>
+    /// Thrown when the renderer receives an invalid event ID that it can't dispatch.
+    /// </summary>
+    public class InvalidEventException : Exception
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="InvalidEventException"/>.
+        /// </summary>
+        /// <param name="eventId">The id of the invalid event.</param>
+        public InvalidEventException(ulong eventId)
+            : this(eventId, $"There is no event handler associated with this event.")
```

Comment:
TBH it's strange to have this constructor overload with a hardcoded message that isn't completely obvious from the signature. Could this overload be `(ulong eventId, string message)`, so the caller specifies the message?

Reply:
Oh in fact I see that overload already exists below. So could this one be removed?

Reply:
I think I'm going to inline the Event ID into the message at the call-sites and be done with it. (Will get rid of the EventId field)

Reply:
I got rid of the EventId

Reply:
I got rid of the type entirely